custom_components/powercalc/data/gledopto/GL-S-005Z/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script.  Controller: hue, Bulbs: 1, Dummy Load: None",
+    "measure_device": "Kasa, KP115(US)",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v0.24.8:docker"
+    },
+    "name": "Gledopto 4W MR16 Dual White and Color 120°",
+    "standby_power": 1.09,
```

Comment:
This standby power seems quite high for me for a Zigbee based light. Could you recheck this?

Reply:
Yeah, these Gledopto bulbs are garbage.  I do not recommend.  I re-measured just now:

![image](https://user-images.githubusercontent.com/21226768/185493701-cbf9553d-5ce6-417a-b566-da4a77dedf22.png)

Reply:
Ah!  I might have it.  The DC power adapter looks like it's pulling a 0.6W load by itself.  This is without the bulb plugged in JUST my DC power supply:

![image](https://user-images.githubusercontent.com/21226768/185494761-ccf356c9-58a5-44e8-801e-f45dce95a6eb.png)

So I think all these measurements should be adjusted downwards by 0.6W, do you agree?

Reply:
That's probably worth doing